etna/core/templatetags/sections_tags.py

```diff
@@ -20,3 +25,20 @@ def jumplinks(page):
     return {
         "sections": sections,
     }
+
+
+@register.filter
+def cookie_usage(record: dict) -> bool:
+    """
+    Return the True/False based on cookie usage value
+    if no cookie set it will return False
+    """
+    try:
+        usage = False
+        if "cookies_policy" in record:
```

Comment:
A little tweak here will cater for when the value is `None`
```suggestion
        if record and "cookies_policy" in record:
```